src/components/vm/disks/diskAdd.jsx

```diff
@@ -543,9 +543,10 @@ export class AddDiskModalBody extends React.Component {
         this.setState({ dialogError: text, dialogErrorDetail: detail });
     }
 
-    onAddClicked() {
+    async onAddClicked() {
```

Comment:
It's the first time we use `async` in c-machines. I don't have anything against it, I just wonder what it gives you here? What's the difference to just returning the spawn promise?

Reply:
As I said, I'm totally fine with using `async`/`await` -- I just don't understand what it does here. *If* you use these, then you shouldn't use `.then()`/`.catch()` on the cockpit.spawn() promise, but just await it and write the whole block as "synchronous" code. *That* would make it look nicer. But merely making the event handler async as seen from the outside seems strange -- does anything in React care?